elytron/src/main/java/org/wildfly/extension/elytron/AuthenticationClientDefinitions.java

```diff
@@ -328,7 +334,16 @@ static ResourceDefinition getAuthenticationClientDefinition() {
                 }
 
                 final Function<AuthenticationConfiguration, AuthenticationConfiguration> finalConfiguration = configuration;
-                return () -> finalConfiguration.apply(null);
+                return () -> {
+                    try {
+                        return finalConfiguration.apply(null);
+                    } catch (IllegalStateException e) {
+                        if (e.getCause() != null) {
+                            throw new StartException(e.getCause().getMessage());
+                        }
+                        throw new StartException(e.getMessage());
```

Comment:
Yes, it is.

I decided to discard the cause because both CLI and server log will print the error message twice, like:
<pre>
{
    "outcome" => "failed",
    "failure-description" => {"WFLYCTL0080: Failed services" => {"org.wildfly.security.authentication-configuration.cfg2" => "WFLYELY00004: Unable to start the service.
    Caused by: org.jboss.as.controller.OperationFailedException: WFLYELY00920: Credential alias \"d\" of credential type \"org.wildfly.security.credential.PasswordCredential\" does not exist in the store [ \"WFLYELY00920: Credential alias \\\"d\\\" of credential type \\\"org.wildfly.security.credential.PasswordCredential\\\" does not exist in the store\" ]"}},
    "rolled-back" => true
}
</pre>

but it should be another issue if you agree, I will add the cause back for the accurate stack trace.